src/preamble.js

```diff
@@ -429,9 +429,9 @@ function exitRuntime() {
 #if EXIT_RUNTIME
   callRuntimeCallbacks(__ATEXIT__);
   <<< ATEXITS >>>
-#if USE_PTHREADS
-  PThread.runExitHandlers();
```

Comment:
Note that I removed this potentially redundant call, since I assume the exit handlers are called when threads are terminated (but I didn't verified this).

Reply:
I'm not sure about this part, we would need to do more check.  Maybe revert it for now and we can see about landing this seperately?runExitHandlers

Reply:
OK, reverted to its original state with commit 651e24e5721caefa9c0bee293f6d7d40333db254.

Reply:
I'm marking this conversation as resolved, now that PR #14512 has landed and this particular change has been reverted to its original state.